drivers/slipdev/slipdev.c

```diff
@@ -245,12 +252,14 @@ static const netdev_driver_t slip_driver = {
     .set = netdev_set_notsup,
 };
 
-void slipdev_setup(slipdev_t *dev, const slipdev_params_t *params)
+void slipdev_setup(slipdev_t *dev, const slipdev_params_t *params, uint8_t index)
 {
     /* set device descriptor fields */
     dev->config = *params;
     dev->state = 0;
     dev->netdev.driver = &slip_driver;
+
+    netdev_register(&dev->netdev, NETDEV_SLIPDEV, index);
```

Comment:
Shouldn't this be wrapped with `MODULE_SLIPDEV_L2ADDR` as well now? As far as I understood you, this is what caused the problem in the first place.

Reply:
Well it's still useful for `gnrc_netif_get_by_type()`